store/MovieQuotesContext.tsx

```diff
@@ -0,0 +1,34 @@
+import React, { createContext, useState } from 'react';
+import { Children, ContextData } from 'types';
+
+export const MovieQuotesContext = createContext({
+  authModalState: false,
+  changeRegistrationModalState: () => {},
+  loginModalState: false,
+  changeLoginModalState: () => {},
+});
+
+export const MovieQuotesContextProvider: React.FC<Children> = (props) => {
```

Comment:
i don't think name of this file is appropriate, movieQuotesContext and u are storing Login and register state here, change name